tests/visual/panelbar-angular.html

```diff
@@ -29,40 +29,40 @@
 
         <section>
             <kendo-panelbar class="k-panelbar">
-                <kendo-panelbar-item class="k-panelbar-header k-item">
+                <kendo-panelbar-item class="k-panelbar-header k-item k-level-0">
```

Comment:
I see no `k-level-0` & `k-level-1` related styles. Why do we need these classes in the rendering?

Also, shouldn't it be `k-panelbar-level-*` ?

Reply:
As discussed, we will keep the `k-level-0` and `k-level-1` classes in the rendering.